kubernetes-manifests/checkoutservice.yaml

```diff
@@ -64,8 +64,6 @@ spec:
             value: "currencyservice:7000"
           - name: CART_SERVICE_ADDR
             value: "cartservice:7070"
-          - name: DISABLE_PROFILER
```

Comment:
Because I implemented the Cloud Ops options in the Helm chart, I took the initiative to update the associated Kubernetes manifest to be accurate with what we have in the repo and not misleading. It's actually `ENABLE_PROFILER` for `checkoutservice` now.